django/db/migrations/questioner.py

```diff
@@ -48,7 +48,7 @@ def ask_initial(self, app_label):
             elif hasattr(migrations_module, "__path__"):
                 if len(migrations_module.__path__) > 1:
                     return False
-                filenames = os.listdir(migrations_module.__path__[0])
+                filenames = os.listdir(list(migrations_module.__path__)[0])
```

Comment:
could [ ] be used instead of list() ?

Reply:
No, wrapping in a list is different from coercing to list.